portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +27,41 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+let turn;
+function initTic(){
+  turn = 'X';
+  var tic = document.getElementById("tic");
+  tic.appendChild(document.createTextNode("AAAA"));
```

Comment:
The text should be changed to something useful or the line removed altogether.

Reply:
I will remove it. It was left from debugging and I forgot about it.